packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -1,7 +1,18 @@
-## AWS Elastic Container Service (ECS) Construct Library
+## Amazon Elastic Container Service (Amazon ECS) Construct Library
```

Comment:
@Doug-AWS we had a chat today about this and decided that we will change all the README files to H1s instead of H2s and use the short name as the H1 heading (#501).

So the title here should be simply:

```
# Amazon ECS
```

copy: @RomainMuller @rix0rrr

Reply:
Great, I wasn't a fan of the long titles anyway. When I drop it into the docs, I'll probably append "Examples" to the H1 heading ("Amazon ECS" -> "Amazon ECS Examples".

Reply:
Let's discuss this idea of "dropping the READMEs into the docs". I don't think it's a sustainable approach. These README files are constantly changing

Reply:
Sure, but keep in mind I use a script. Whenever we release, I run the script and snarf the content of the L2 README.md files in "Developer Preview" status into the guide's Examples section. It takes me all of about 5 minutes. The biggest effort is editing the L2 README.md file once the status is "Developer Preview". That's a much longer process.

Reply:
But those are not examples, so why would we put them there?

Reply:
How about in an "About the AWS Construct Libraries" section? I can easily change my script.